### PR TITLE
Added CHANGELOG note about translated property

### LINKED ISSUE #606: Add CHANGELOG note about the translated custom propertyname fix.

### PR DESCRIPTION
**Pull request by dantleech**
Added changelog note for custom translated property mapping fix. Fixes #606

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -14,6 +14,8 @@ Changelog
 * **2015-04-07**: Class metadata now validates that you can not map the UUID on
                   Documents that are not referenceable. Either set your
                   documents `referenceable=true` or remove the UUID mapping.
+* **2015-03-04**: Translated properties now take into account the "property"
+                  mapping (allowing customiation of the PHPCR property name).
```
Comment (dbu) [inline CHANGELOG.md]: s/customiation/customization

**Comment by dbu** [comment]
thanks! can you please fix the typo?

File: CHANGELOG.md (hunk 1 above)
Comment (dantleech) [inline CHANGELOG.md]: Will fix and merge  next chance I have

On 6 July 2015 09:28:47 CEST, David Buchmann <notifications@github.com> wrote:
>> @@ -14,6 +14,8 @@ Changelog
>>  * **2015-04-07**: Class metadata now validates that you can not map
>the UUID on
>>                    Documents that are not referenceable. Either set
>your
>>                    documents `referenceable=true` or remove the UUID
>mapping.
>> +* **2015-03-04**: Translated properties now take into account the
>"property"
>> +                  mapping (allowing customiation of the PHPCR
>property name).
>
>s/customiation/customization
>
>---
>Reply to this email directly or view it on GitHub:
>https://github.com/doctrine/phpcr-odm/pull/647/files#r33910874

-- 
Sent from my Android device with K-9 Mail. Please excuse my brevity.

Comment (dbu) [inline CHANGELOG.md]: thanks

**Pull request merged**